lib/unit-ruby/util/api_resource.rb

```diff
@@ -101,6 +101,45 @@ def self.belongs_to(resource_name, class_name: nil)
       end
     end
 
+    def self.has_many(resource_name, class_name: nil)
+      class_name ||= resource_name.to_s.camelize
+
+      define_method(resource_name) do
+        singular_resource_name = resource_name.to_s.singularize.to_sym
+
+        # if there is only one of the resources, Unit will return the
+        # singular name of that relationship in their response.
+        # For example, if a deposit account has one customer, `customer`
+        # will be returned from Unit as a resource, whereas if there
+        # are more than one customer, then customers will be returned from Unit
+
+        # the below will translate these singular relationship
+        # into a plural relationship - ie `customer` -> `customers`
+
+        if relationships.keys.include?(singular_resource_name)
+
+          relationship_id = relationships.dig(singular_resource_name, :data, :id)
+
+          return nil unless relationship_id
+
+          Kernel.const_get(class_name).find_all([relationship_id])
+
+        else
+          relationship_ids = relationships.dig(resource_name.to_sym, :data).map { |x| x[:id] }
+
+          return nil if relationship_ids.empty?
+
+          Kernel.const_get(class_name).find_all(relationship_ids)
+        end
+      end
+
+      define_method("#{resource_name}=") do |resource|
```

Comment:
Does this need to accept an array of resources?

I would have expected usage to look like:

```
deposit_account = DepositAccount.find(...)
deposit_account.customers = [customer_1, customer_2]
deposit_account.save
```

Reply:
Possibly? In taking a look at this comment & playing around with this a little, I'm not a little unsure what this is doing. Would you be up for pairing on this a little tomorrow AM?